org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -231,7 +231,7 @@ Instantiation:
 Connection:
     ((leftPorts += VarRef (',' leftPorts += VarRef)*)
     | ( '(' leftPorts += VarRef (',' leftPorts += VarRef)* ')' isIterated ?= '+'?))
-    ('->' | physical?='~>') 
+    ('->' | physical?='~>' | cross?='-x->') 
```

Comment:
Suggest we change the syntax as discussed to be a modifier on VarRef to optionally reference a bank of multiports as `interleaved(b.p)`.